pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.21</version>
+    <version>2.30</version>
```

Comment:
Well, predated that POM. I did not bother to update it further at this time; surely that is not required.